applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -634,6 +609,22 @@ int slm_at_host_init(void)
 		LOG_ERR("Cannot bind UART device\n");
 		return -EINVAL;
 	}
+	/* Save UART configuration to setting page */
+	if (!uart_configured) {
+		err = uart_config_get(uart_dev, &slm_uart);
+		if (err != 0) {
+			LOG_ERR("uart_config_get: %d", err);
+			return err;
+		}
+		err = slm_setting_uart_save();
+		if (err != 0) {
+			LOG_ERR("uart_config_get: %d", err);
+			return err;
+		}
+		uart_configured = true;
+	} /* else re-config UART based on setting page */
```

Comment:
Is this some kind of TODO comment? It doesn't seem to match the implementation.

Reply:
Yes, Larry will create a PR to allow re-configurate UART interface by AT command, and configurations made persistent by setting page.